src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -54,13 +93,15 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   const containerCssVars = useCss({
     "--event--margin-left": `${eventMarginLeftPx}px`,
     "--event--width": `${eventWidthPx}px`,
+    "--event--widen": `${expandedEventPx}px`,
```

Comment:
'Widen' seems weird here.. would this work?

```suggestion
    "--event--expanded-width": `${expandedEventPx}px`,
```

Reply:
done f718762